CHANGELOG.md

```diff
@@ -21,9 +31,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 ### Added
 - Add support for OpenID-Connect [(#272)](https://github.com/ManageIQ/manageiq-appliance-build/pull/272)
 
-## Unreleased as of Sprint 86 ending 2018-05-21
+## Gaprindashvili-4 - Released 2018-07-16
```

Comment:
Yes, I believe Satoe (it might have been someone else) pointed this out to me that PR268 was already released in Gap-4.

Reply:
Understood. But you have to understand how tedious these things are to do. If this were code, then I'd fully agree, but we've sometimes had "running PRs" with multiple sprints in a single PR.

Reply:
I do understand how tedious this is, but even this discussion would have been avoided if you had made this a separate commit, or at least called it out in the PR title or description. As it is, this looks like it was included by mistake.

We can merge this as is this time around, but only because this conversation serves as documentation for why this is being done.

Next time, if a separate PR is too much, then at least make this a separate commit and call out the additional change in the PR description.